mbodied/agents/sense/object_detection_agent.py

```diff
@@ -50,12 +59,18 @@ def act(self, image: Image, objects: list[str] | str, *args, api_name: str = "/d
 
         if isinstance(objects, list):
             objects = ",".join(objects)
-        annotated_img, json_dict = self.actor.predict(image.base64, objects, *args, api_name=api_name, **kwargs)
+        annotated_img, json_dict = self.actor.predict(
+            image.base64, objects, model_type=model_type, *args, api_name=api_name, **kwargs
+        )
         return World.model_validate(json_dict)
 
 
 # Example usage:
 if __name__ == "__main__":
     agent = ObjectDetectionAgent(model_src="https://api.mbodi.ai/sense/")
-    result = agent.act(image=Image("resources/bridge_example.jpeg", size=(224, 224)), objects=["spoon", "bowl"])
+    result = agent.act(
+        image=Image("resources/bridge_example.jpeg", size=(224, 224)),
```

Comment:
files("mbodied") / resources

Reply:
The resources is in root directory. Don't want to move them all yet. Lots of dependencies. They should be in separate PR.